### ISSUE TITLE
torch: train_ProTACT.py validation loss function 실행 방법

### ISSUE THREAD

**Issue by jwanp** [2024-SKKU-AI-project-6/ProTACT#1]
## validation loss function 실행 방법
torch/train_ProTACT.py 에서 모델 학습 부분
validation loss 측정할때 두번째 거로 하면 될까요? 원래는 첫번째거로 되어 있음
- loss = criterion(outputs, targets)
- loss = criterion.loss_function(outputs, targets)

train loss function:
- loss = criterion.loss_function(outputs, targets)

전체 코드
```python
for epoch in range(epochs): # 50
    print(f'Epoch {epoch + 1}/{epochs}')
    start_time = time.time()

    # train
    model.train()
    train_loss = 0.0
    for batch_data in train_loader:
        optimizer.zero_grad()
        batch_data = [x.to(device) for x in batch_data]
        inputs, targets = batch_data[:-1], batch_data[-1]
        outputs = model(*inputs)
       # 이부분하고
        loss = criterion.loss_function(outputs, targets)
        loss.backward()
        optimizer.step()
        train_loss += loss.item() * batch_data[0].size(0)
    train_loss /= len(train_loader.dataset)

    # validate
    model.eval()
    val_loss = 0.0
    with torch.no_grad():
        for batch_data in dev_loader:
            batch_data = [x.to(device) for x in batch_data]
            inputs, targets = batch_data[:-1], batch_data[-1]
            outputs = model(*inputs)
            # 이부분
            #loss = criterion(outputs, targets)
            loss = criterion.loss_function(outputs, targets)
            val_loss += loss.item() * batch_data[0].size(0)
        val_loss /= len(dev_loader.dataset)

    custom_hist.update(train_loss, val_loss)

    # evaluate
    tt_time = time.time() - start_time
    print(f"Training one epoch in {tt_time:.3f} s")
    evaluator.evaluate(model, epoch + 1)
    print(f"Train Loss: {train_loss:.4f} || Val Loss: {val_loss:.4f}")

evaluator.print_final_info()
```
